formula/src/main/java/com/instacart/formula/internal/DeferredTransition.kt

```diff
@@ -11,11 +9,10 @@ import com.instacart.formula.Transition
  */
 class DeferredTransition<Input, State, EventT> internal constructor(
     private val listener: ListenerImpl<Input, State, EventT>,
-    private val transition: Transition<Input, State, EventT>,
```

Comment:
This is more of a potential bug fix: we should use mutable `transition` that lives in `ListenerImpl` instead relying on fixed value. With dispatching, this value could be out of date by the time we call `execute`